consensus/polybft/state.go

```diff
@@ -181,6 +187,21 @@ func decodeEventData(eventDataMap map[string]interface{}, log *ethgo.Log,
 	return eventCreator(id, sender, receiver, data), nil
 }
 
+// convert converts types.Log to ethgo.Log
+func convert(log *types.Log) *ethgo.Log {
+	l := &ethgo.Log{
+		Address: ethgo.Address(log.Address),
+		Data:    log.Data,
+		Topics:  make([]ethgo.Hash, len(log.Topics)),
+	}
+
+	for i := 0; i < len(log.Topics); i++ {
+		l.Topics[i] = ethgo.Hash(log.Topics[i])
+	}
```

Comment:
Micro optimization :slightly_smiling_face: 

```suggestion
	for i, topic := range log.Topics {
		l.Topics[i] = ethgo.Hash(topic)
	}
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/852/commits/2905c39f46bf6a7af6d323b1e016924bb55a3573